### PR TITLE
Accessibility checks

### PR DESCRIPTION
**Pull request by Gambit142**
In this branch feature, we used the W3C Web Accessibility Initiative (WAI) template to check if this portfolio meets all the requirements.

The WAI requirements we checked for are:

- Page Titles https://github.com/Gambit142/Template-Portfolio/blob/73d5153e9a883b41ed6e54d6131bed330d23563c/index.html#L6
- Image Text Alternatives - No issues found
- Text Headings - No issues found
- Color Contrast https://github.com/Gambit142/Template-Portfolio/blob/73d5153e9a883b41ed6e54d6131bed330d23563c/styles.css#L330 
https://github.com/Gambit142/Template-Portfolio/blob/73d5153e9a883b41ed6e54d6131bed330d23563c/styles.css#L726
- Resize https://github.com/Gambit142/Template-Portfolio/blob/73d5153e9a883b41ed6e54d6131bed330d23563c/styles.css#L111-L180
- Interaction - No issues found
- Moving Content - No issue found
- Multimedia - No issues found
- Basic Structure of Page - No Issues found

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Gambit142, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -158,7 +159,7 @@ <h1 class="style">My recent work</h1>
     </div>
```
Comment (mke2111) [inline index.html:159]: - [ ] It is a better practice to have one `h1` tag per page. This helps with SEO. For your page, your name should keep the `h1` tag. My recent work should be `h2` tag and then the project title should have `h3` tag. Ensure to maintain a meaningful hierarchy of heading levels. Make all the changes for your `hx` tags

File: index.html [hunk 2]
```diff
@@ -253,35 +262,50 @@ <h2 class="style">Multi-Post Stories</h2>
...
     <div class="footer-image-div">
-      <ul class="display-flex justify-content">
-        <li>
+      <ul class="social-icons display-flex justify-content">
+        <li class="icon-frame">
           <a href="#"><img src="Images/Icon-GitHub.png" alt="A Github logo Icon"></a>
         </li>
-        <li>
+        <li class="icon-frame">
           <a href="#"><img src="Images/Icon-Linkedin.png" alt="A LinkedIn logo Icon"></a>
```
Comment (mke2111) [inline index.html]: OPTIONAL

- [ ] Keep the alternative text a summary of the img tag. For this case, Linkedin logo is just enough. You may want to keep it simple for all the alt text.

File: index.html [hunk 3]
```diff
@@ -1,22 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <title>Mobile Version</title>
+  <title>Francis Ugorji's Template Portfolio</title>
```
Comment (mke2111) [inline index.html]: - [ ] Remove the word template from the page title. It is much better and meaningful without it

**Comment by Gambit142** [comment]
Hi @mke2111,

Thank you for the feedback, it was useful as I learned from it. We have made the suggested changes and they work perfectly. 👍

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Gambit142 and @see-why, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**